src/index.ts

```diff
@@ -1,2 +1,13 @@
 export { rendererRegistryEntries, cellRegistryEntries } from "./renderers"
 export type * from "./ui-schema"
+
+export { BooleanControl } from "./controls/BooleanControl"
+export { AlertControl } from "./controls/AlertControl"
```

Comment:
As part of these changes, can you rename this to AlertLayout (I think that's the best name, but it's possible this is best described as a cell) ... getting names right is crucial bc changing names afterward is a breaking change

Reply:
Makes sense to me! ✅